### PR TITLE
VFX CY2025 compliance

### PR DESCRIPTION
**Pull request by Meakk**
Fix https://github.com/f3d-app/f3d/issues/1779
See https://vfxplatform.com/

Changing GLIBC might be trickier since we need to change our Docker base image to use a RH based distro.

### CODE REVIEW

File: .github/actions/alembic-install-dep/action.yml [hunk 1]
```diff
@@ -10,20 +10,21 @@ runs:
   using: "composite"
   steps:
 
+    # Alembic is part of VFX reference platform (CY2025: 1.8.x)
```
Comment (mwestphal) [inline .github/actions/alembic-install-dep/action.yml:13]: if you put comment like that, lets add such comment everywhere it applies

Comment (mwestphal) [reply .github/actions/alembic-install-dep/action.yml:13]: (not the osx stuff, but the dep versions)

File: .github/actions/assimp-install-dep/action.yml [hunk 2]
```diff
@@ -15,15 +15,15 @@ runs:
       uses: actions/cache@v4
       with:
         path: dependencies/assimp_install
-        key: assimp-v5.4.2-${{runner.os}}-${{inputs.cpu}}-0
+        key: assimp-v5.4.3-${{runner.os}}-${{inputs.cpu}}-0
```
Comment (mwestphal) [inline .github/actions/assimp-install-dep/action.yml:18]: missing VFX version comment

File: .github/actions/blosc-install-dep/action.yml [hunk 3]
```diff
@@ -15,7 +15,7 @@ runs:
       uses: actions/cache@v4
       with:
         path: dependencies/blosc_install
-        key: blosc-1.21.5-${{runner.os}}-${{inputs.cpu}}-5
+        key: blosc-1.21.6-${{runner.os}}-${{inputs.cpu}}-0
```
Comment (mwestphal) [inline .github/actions/blosc-install-dep/action.yml:18]: missing VFX version comment

File: .github/actions/draco-install-dep/action.yml [hunk 4]
```diff
@@ -15,7 +15,7 @@ runs:
       uses: actions/cache@v4
       with:
         path: dependencies/draco_install
-        key: draco-1.5.7-${{runner.os}}-${{inputs.cpu}}-2
+        key: draco-1.5.7-${{runner.os}}-${{inputs.cpu}}-3
```
Comment (mwestphal) [inline .github/actions/draco-install-dep/action.yml:18]: missing VFX version comment

File: .github/actions/imath-install-dep/action.yml [hunk 5]
```diff
@@ -15,7 +15,7 @@ runs:
       uses: actions/cache@v4
       with:
         path: dependencies/imath_install
-        key: imath-v3.1.11-${{runner.os}}-${{inputs.cpu}}-1
+        key: imath-v3.1.12-${{runner.os}}-${{inputs.cpu}}-0
```
Comment (mwestphal) [inline .github/actions/imath-install-dep/action.yml:18]: missing VFX version comment

File: .github/actions/pybind11-install-dep/action.yml [hunk 6]
```diff
@@ -9,7 +9,7 @@ runs:
       uses: actions/cache@v4
       with:
         path: dependencies/pybind11_install
-        key: pybind11-v2.12.0-${{runner.os}}-0
+        key: pybind11-v2.13.6-${{runner.os}}-0
```
Comment (mwestphal) [inline .github/actions/pybind11-install-dep/action.yml:12]: missing VFX version comment

**Comment by Meakk** [comment]
Closed all discussions about the VFX comments, as per our discussion on discord

**Review by mwestphal — APPROVED** [review]

**Pull request merged**